tests/system/action/meeting/test_create.py

```diff
@@ -167,6 +166,47 @@ def test_create_check_users(self) -> None:
             "user/1", {f"group_${meeting['id']}_ids": [admin_group_id]}
         )
 
+    def test_create_with_request_user(self) -> None:
+        self.set_models(
```

Comment:
what do you think about a comment for this test, explaining group2 as admin group for the request user?

Reply:
Sure, I can add a comment